DEVELOP.rst

```diff
@@ -71,10 +75,14 @@ node, like so::
 This  ``base_uri`` parameter is intended for development use only.
 
 
-Adding New Dependencies
+**********************
+Development operations
+**********************
+
+Adding new dependencies
 =======================
 
-Application dependencies should be added as follow::
+In order to add application dependencies, just invoke::
```

Comment:
I recommend against using "just", as it does have a slightly patronising tone.

Try:

"To add application dependencies, invoke::"